release-team/role-handbooks/release-lead.md

```diff
@@ -68,7 +68,11 @@ A guiding principle for the lead is be an arbiter of decisions, and not the prim
 
 * Feature exceptions are to be reviewed by the owner SIG and brought to the release team for assessment of risk, especially across SIGs
 
-* All issues in the milestone are considered release-blocking until proven otherwise by the owner SIG
+* All issues and PRs in the milestone are considered release-blocking until proven otherwise by the owner SIG.
+    * Issues and PRs are added to the milestone only by members of the [kubernetes-milestone-maintainers" GitHub team]((https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers) which primarily includes SIG leads.
+    * Typically the release lead will have been granted maintainer priviledgeon this team for adding/removing folks as needing during the release cycle if SIG leads have turned over.  If not and it is needed ask in Slack #sig-contribx to find the current administrator of teams.
+    * The release lead may also need to add members of the release team, and should check with and prune prior release team members who are no longer active.
```

Comment:
Can only the release lead add/prune members to the milestone-maintainers list? Also is there a quick overview of how to add members?